cfme/web_ui/__init__.py

```diff
@@ -1280,6 +1280,7 @@ def _fill_form_list(form, values, action=None, action_always=False):
 
     """
     logger.info('Beginning to fill in form...')
+    sel.wait_for_ajax()
```

Comment:
There is some kind of a nasty script (?) in CFME: after the form is loaded the focus is reset to hostname. As a result it tries to validate host 'ootcfme-01...' with user 'r' and correct password. 

Not sure why is happening, but it fixes the problem